### DISCUSSION TITLE
Gopro Hero 2024 (4k) - Gopro Labs?

### DISCUSSION THREAD

**Discussion by mirankevic** [gopro/labs#1100] [General]
Good day,

I was wondering if its possible to install gopro labs on the new gopro 4k 2024 that I got today? There are no video settings and I would like to change the settings as I was thinking of using this gopro on top of my fpv drone.

Thanks

**Comment by dnewman-gpsw** [comment]
No plans for Labs on HERO 2024. It is a different platform, making any Labs port difficult/time-consuming, ultimately resulting in limited abilities. And this would tradeoff Labs extensions for Black series cameras.  So for these reasons, there is no Labs for the entry level camera. Same as the past, no Labs on Silver and White addition cameras.

This could change in the future, so your interest is noted.

**Reply by jsolaun** [reply]
Many FPV drone pilots would be **highly interested** in this camera due to its light weight. In fact we have been waiting for a camera like this since the Hero 5 Session. The only thing holding us back is the incapability to change the default settings such as the aspect ratio. 16:9 is not very convenient for stabilization, and this simple setting cannot be changed.

**Comment by mirankevic** [comment]
I would highly recommend gopro invests in this as this camera would be perfect for fpv. Are ND filters coming for this camera? Thanks, Miran Sent from my iPhoneOn 26 Sep 2024, at 20:41, David Newman ***@***.***> wrote:﻿
No plans for Labs on HERO 2024. It is a different platform, making any Labs port difficult/time-consuming, ultimately resulting in limited abilities. And this would tradeoff Labs extensions for Black series cameras.  So for these reasons, there is no Labs for the entry level camera. Same as the past, no Labs on Silver and White addition cameras.
This could change in the future, so your interest in noted.

**Reply by dnewman-gpsw** [reply]
Labs is not the only way, that only serves a few percentage of users. You are only seeing launch firmware, I'm sure there is more coming (I'm not on the HERO team, so don't have any details.)

**Comment by mirankevic** [comment]
I see, if I can’t change the settings and ND filters aren’t coming I’ll return the gopro. gopro would have dropped the ball on not having this as gopro 11 mini camera On 26 Sep 2024, at 23:27, David Newman ***@***.***> wrote:﻿
Labs is not the only way, that only serves a few percentage of users. You are only seeing launch firmware, I'm sure there is more coming (I'm not on the HERO team, so don't have any details.)

**Reply by dnewman-gpsw** [reply]
You are likely posting to the the wrong forum. I thought you where asking about Labs firmware. Labs doesn't release ND filters, our choose menu settings, etc.

**Comment by mirankevic** [comment]
So is there any other ways to change the settings besides Labs? You
mentioned there were other ways, I only know of labs.

Thanks David,



On Thu, Sep 26, 2024 at 11:43 PM David Newman ***@***.***>
wrote:

> You are likely posting to the the wrong forum. I thought you where asking
> about Labs firmware. Labs doesn't release ND filters, our choose menu
> settings, etc.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gopro/labs/discussions/1100#discussioncomment-10768565>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQAJZ7B3AWDKBWMYRFK27FDZYR5YHAVCNFSM6AAAAABO5TE4F6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANZWHA2TMNI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Miran Kevic
Director/Sr. motion designer
T: +47 99278216
W:http://www.mirankevic.com

**Comment by mirankevic** [comment]
So is there any other ways to change the settings besides Labs? You
mentioned there were other ways, I only know of labs.

Thanks David,

On Thu, Sep 26, 2024 at 11:43 PM David Newman ***@***.***>
wrote:

> You are likely posting to the the wrong forum. I thought you where asking
> about Labs firmware. Labs doesn't release ND filters, our choose menu
> settings, etc.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gopro/labs/discussions/1100#discussioncomment-10768565>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQAJZ7B3AWDKBWMYRFK27FDZYR5YHAVCNFSM6AAAAABO5TE4F6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANZWHA2TMNI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Miran Kevic
Director/Sr. motion designer
T: +47 99278216
W:http://www.mirankevic.com

**Comment by dnewman-gpsw** [comment]
Labs isn't on that camera, so settings, whatever there are, would be in menus and Quik.

**Comment by mirankevic** [comment]
There is nothing, just photo, video and slow-mo. No options
whatsoever...Gopro really dropped the ball on this one.

I told them to tell SR management that this camera could have been a great
fpv camera as it's small, light, and fixed heating issues.

They could have made a ton of money on FPV pilots as most of us would buy
this camera with ND filters.

What a huge mistake to make a GoPro without any possibility to change any
settings...Huge mistake.

I will be returning the Gopro tomorrow, if they change this on the camera I
will buy it again but I doubt they will.

M

On Fri, Sep 27, 2024 at 12:55 AM David Newman ***@***.***>
wrote:

> Labs isn't on that camera, so settings, whatever there are, would be in
> menus and Quik.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gopro/labs/discussions/1100#discussioncomment-10768909>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQAJZ7A7YGTRYDBUC23GEOTZYSGFFAVCNFSM6AAAAABO5TE4F6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANZWHA4TAOI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Miran Kevic
Director/Sr. motion designer
T: +47 99278216
W:http://www.mirankevic.com

**Reply by dnewman-gpsw** [reply]
Sounds good.

**Comment by kouyoumin** [comment]
I once considered buying a hero as a dashcam, but I ended up getting the 11 mini, which is better and cheaper. (and labs capable)
